src/app/home/home.component.ts

```diff
@@ -28,7 +28,7 @@ export class HomeComponent implements AfterViewInit {
     displayWriteTextOrUploadDocumentFlag: any = true;
     characterCount: number = 0;
     wordCount: number = 0;
-    innerHTMLOfEditor: any = this.LINE_BROKEN_PARAGRAPH;
```

Comment:
This and then the following `document.getElementById(this.EDITOR_KEY)!.innerHTML = this.LINE_BROKEN_PARAGRAPH;` lines should not be changed. A solution must be tailored with these in mind.

Reply:
Why it should not be changed? Please share with us some cases where this doesn't work or why it is unacceptable. Which other code parts depend on it being like that and not otherwise. I think it would nice to [explain the why](https://google.github.io/eng-practices/review/reviewer/comments.html#why) here. Helps novice contributors make a good headstart. 

As such issues cannot be alleviated otherwise, such decisions and unacceptable code changes should be noted somewhere that is accessible to all. The code comments are a good candidate for this. I created [an issue](https://github.com/OpenCovenant/quill/issues/89) to track and avoid such issues.

Reply:
Having the `innerHTML` to be `<p><br></p>` means that during interactions with the editor (the `div` that contains this `innerHTML`) will work with paragraphs (`<p>`s). E.g. pressing Enter in the editor will change its `innerHTML` to `<p><br></p><p><br></p>`, additionally typing "something" in it and pressing Enter and then typing something "else" would change it to `<p>something</p><p>else</p>`. Having this "paragraphized" behavior is aligned with what the user expects in regards to the writing of a text.

Having more comments for now is definitely a good idea, I've created [this PR](https://github.com/OpenCovenant/quill/pull/94) and put there some changes, let me know what you'd like to see added/changed.